CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
```

Comment:
Can you convert our old CHANGELOG to CHANGELOG.md.

Example: https://github.com/edestecd/puppet-mariadb/commit/0fd43c12bb3ce0e0ff2d098ddf7e6560b0ef3416